src/test/compile-fail/symbol-names/basic.rs

```diff
@@ -7,17 +7,10 @@
 // <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
-//
-// error-pattern:already defined
-
 
-#![allow(warnings)]
+#![feature(rustc_attrs)]
 
+#[rustc_symbol_name] //~ ERROR _ZN5basic4main
```

Comment:
This name and others should end with an 'E' to close the `nested name` that was started with the 'N'.